rviz_rendering/CMakeLists.txt

```
@@ -27,6 +27,7 @@ else()
 endif()
 
 find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_ros REQUIRED)
```

We don't typically need both `ament_cmake_ros` and `ament_cmake`; the former depends on the latter.

```suggestion
find_package(ament_cmake_ros REQUIRED)
```